Quora/static/css/index.css

```diff
@@ -10,11 +10,11 @@ body {
     background-size: cover;
     width: 100%;
     height: 100%;
-    position: fixed;
     color: #333;
     font-size: 13px;
     background-color: rgb(250, 250, 250);
     font-family: 'Overpass';
+    position: absolute;
```

Comment:
```suggestion
```
This should be `position: relative`, but since that's the default value there's no need to set it explicitly.
`position: absolute` makes the element stick to a defined position in the page, removing it from the default flow of the DOM.